powerauth-webflow-authentication/src/main/java/io/getlime/security/powerauth/lib/webflow/authentication/controller/AuthMethodController.java

```diff
@@ -174,7 +174,8 @@ protected String authenticate(T request) throws E {
      * @throws AuthStepException In case authorization fails.
      */
     protected UpdateOperationResponse authorize(String operationId, String userId) throws NextStepServiceException, AuthStepException {
-        return authorize(operationId, userId, null);
+        UpdateOperationResponse response = authorize(operationId, userId, null);
```

Comment:
This will show a warning "... can be simplified", did you forget to add log here?

Reply:
I added a log message, later found out it was duplicate and then removed it. I will simplify the return call.